iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -624,8 +625,11 @@ static LogicalResult legalizeDispatchWorkgroupOperands(
     Value bbArg = block.getArguments().back();
     Value repl = bbArg;
     if (bbArg.getType().isa<IREE::Flow::DispatchTensorType>()) {
-      repl = b.create<IREE::Flow::DispatchTensorLoadOp>(loc, operand.getType(),
-                                                        bbArg);
+      repl = b.create<IREE::Flow::DispatchTensorLoadOp>(
+          loc, operand.getType(), bbArg, ArrayRef<Value>({}),
+          ArrayRef<Value>({}), ArrayRef<Value>({}), b.getI64ArrayAttr({}),
```

Comment:
can you add a builder to the op that defaulted these? this is pretty nasty to try to get right

Reply:
Yes, I am looking into this Interestingly there is a build for `b.create<IREE::Flow::DispatchTensorLoadOp>(loc, operand.getType(), bbArg)`  but it creates an op that crashes because `$source is Nil` might be incorrectly parsing due to a bug in the interface ?!

Reply:
Thats weird, the source should not be nil right.

Nit: You can also just do `ArrayRef<Value>()` instead of `ArrayRef<Value>({})`